### DISCUSSION TITLE
sherlock

### DISCUSSION THREAD

**Discussion by elishaluel** [sherlock-project/sherlock#1254] [Q&A] [answered]
how can I install sherlock in phyton

**Comment by M-warrior** [comment]
hi friends

**Comment by M-warrior** [comment]
can anyone help me?

**Comment by M-warrior** [comment]
how to install this process

**Comment by NguyenTrongTin1908** [comment]
Read it: https://www.sherlock.stanford.edu/docs/software/using/python/

**Comment by SethFalco** [comment, marked as answer]
Installation steps are shown on the README:
https://github.com/sherlock-project/sherlock#installation=

You just need to install Git and Python first if you don't have them already.
If you're on Debian based distro, you can just do:

```sh
sudo apt-get install git python3 python3-pip
```

Other distros will have a similar command, but the package names may differ.

If you're on Windows you can just download it from following websites:
* https://gitforwindows.org/
* https://www.python.org/downloads/windows/

You can confirm the following commands work without error:
```sh
git --version
```

```sh
# If you're on Linux
python3 --version

# If you're on Windows
python --version
```

You should be able to follow the instructions as-is.

Later I'll make a PR proposing this information be added.
